### PR TITLE
Moving Mock Web Services Docs to Wiki

### PR DESCRIPTION
**Pull request by gwendolyngoetz**
### Description

This is now at https://github.com/civiform/civiform/wiki/Mock-Web-Services

more detail is needed to communicate to partners the scope of the PR's changes, use this release notes section.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)

### CODE REVIEW

**Review by caitlinshk — APPROVED** [review]
PR looks good!

Quick question on the wiki page: Should we name the page something like "Mock Web Services for ESRI Address Correction and Service Area Validation" so that it's a bit easier to Ctrl+F for? If someone is looking for how we handle address correction locally, they may not think to Ctrl+F for "mock web services". 

I'm also planning to add more documentation around address correction & service address validation, so maybe we could name the page just "ESRI Address Correction and Service Area Validation" and I can add additional documentation on how the non-mock address correction works?

**Comment by gwendolyngoetz** [comment]
> PR looks good!
> 
> Quick question on the wiki page: Should we name the page something like "Mock Web Services for ESRI Address Correction and Service Area Validation" so that it's a bit easier to Ctrl+F for? If someone is looking for how we handle address correction locally, they may not think to Ctrl+F for "mock web services".
> 
> I'm also planning to add more documentation around address correction & service address validation, so maybe we could name the page just "ESRI Address Correction and Service Area Validation" and I can add additional documentation on how the non-mock address correction works?

No, that's too wordy and won't fit well in the sidebar. I updated the headings which will be enough for searching.

- Esri Endpoints
  - Address Correction
  - Service Area Validation
